UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -5,9 +5,114 @@ import UIKit
 
 /// ViewController
 class ViewController: UIViewController {
```

Comment:
```suggestion
final class ViewController: UIViewController {
```
Название контроллера не отражает его функцию